addons/l10n_in/wizard/l10n_in_withhold_wizard.py

```diff
@@ -138,6 +157,44 @@ def _compute_l10n_in_withholding_warning(self):
                 }
             wizard.l10n_in_withholding_warning = warnings
 
+    @api.depends('related_move_id', 'related_payment_id')
+    def _compute_tax_id(self):
+        for wizard in self:
+            sections = wizard.related_move_id._get_l10n_in_tds_tcs_applicable_sections()
+            if sections:
+                accounts_by_section = {}
+                for line in wizard.related_move_id.line_ids:
+                    section = line.account_id.l10n_in_tds_tcs_section_id
+                    if section in sections:
+                        accounts_by_section[section] = line.account_id
```

Comment:
What if there are multiple accounts in a section?

Reply:
In accounts_by_section, we will store every section that will be applicable, and then check for the last tax used in tds entry for the combination of account, tax, and pan entity, and once we find the tax, it will be set as the default tax in the wizard